spharpy/ambi.py

```diff
@@ -0,0 +1,274 @@
+from pyfar import Signal
+import numpy as np
+from spharpy.spherical import SphericalHarmonics
+import warnings
+
+
+class AmbisonicsSignal(Signal):
+    """Class for ambisonics signals.
+
+    Objects of this class contain data which is directly convertible between
+    time and frequency domain (equally spaced samples and frequency bins). The
+    data is always real valued in the time domain and complex valued in the
+    frequency domain.
+
+    """
+    def __init__(
+            self,
+            data,
+            sampling_rate,
+            n_max,
+            basis_type,
+            normalization,
+            channel_convention,
+            condon_shortley,
```

Comment:
Maybe we can check here if this can be generalized to cover arbitrary phase conventions depending if the basis type is complex or real
```suggestion
            phase_convention='default',
```